src/balatro_gym/cards/joker/joker.py

```diff
@@ -314,6 +316,124 @@ def get_multiplication(
         return board.num_joker_slots - num_jokers + num_negative
 
 
+class Fibonacci(JokerBase):
+    _cost: int = 8
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.UNCOMMON
+
+    def get_mult_card(self, card: PlayingCard, blind: BlindState, board: "BoardState") -> int:
+        target_ranks = [Rank.ACE, Rank.TWO, Rank.THREE, Rank.FIVE, Rank.EIGHT]
+        return 8 if card.rank in target_ranks else 0
+
+
+class SteelJoker(JokerBase):
+    _cost: int = 7
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.MULTIPLICATIVE
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.UNCOMMON
+
+    def get_multiplication(
+        self, scored_cards: Sequence[PlayingCard], blind: BlindState, board: BoardState, scored_hand: PokerHandType
+    ) -> float:
+        n_steel_cards = len([card for card in board.deck.cards if isinstance(card.enhancement, SteelCard)])
+        return 1. + n_steel_cards * 0.2
+
+
+class ScaryFace(JokerBase):
+    _cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.CHIPS
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_chips_card(self, card: PlayingCard, blind: BlindState, board: "BoardState") -> int:
+        has_pareidolia = any([isinstance(j, Pareidolia) for j in board.jokers])
```

Comment:
`PlayingCard.is_face_card()` considers if `Pareidolia` is in play, so I don't think it's needed here.

Reply:
I need to pass the `has_pareidolia` in the params due to the reason above